test/models/ViewModels/VatCertificateViewModelSpec.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.ViewModels
+
+import java.time.LocalDate
+
+import common.TestModels.customerInformationMin
+import models.Address
+import models.viewModels.VatCertificateViewModel
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+
+class VatCertificateViewModelSpec extends AnyWordSpecLike with Matchers {
+
+  "The VatCertificateViewModel .fromCustomerInformation()" when {
+
+    lazy val model = VatCertificateViewModel
+
+    "the customer is missing a first or last name" should {
```

Comment:
Could you also put in a test for the happy path here, i.e. the user has first and last name? I'm guessing it wasn't flagged up in the report due to an unrelated test indirectly covering that branch of code, but it makes sense to have an explicit one for it here.